packages/kit/src/exports/vite/build/build_service_worker.js

```diff
@@ -64,16 +63,16 @@ export async function build_service_worker(
 	);
 
 	await vite.build({
-		base: assets_base(kit),
```

Comment:
I'm a bit suspicious that we were using the base before and are not now.

Reply:
we use `location.href`, so the service worker is portable. this is more robust than letting vite hard-code stuff